netbout-web/src/main/java/com/netbout/rest/MarkdownTxtmark.java

```diff
@@ -47,10 +47,18 @@
         // @checkstyle LineLengthCheck (1 line)
         "(?<!\\]\\()(?<!=\")(https?:\\/\\/[a-zA-Z0-9-._~:\\?#@!$&'*+,;=%\\/]+[a-zA-Z0-9-_~#@$&'*+=%\\/])(?![\\w.]*\\]\\()"
     );
+    /**
+     * New lines detection pattern.
+     */
+    private static final Pattern NEW_LINE = Pattern.compile(
+        "^[ ]{0,3}\\S*[ ]?$", Pattern.MULTILINE
```

Comment:
@dmzaytsev how this pattern is supposed to detect new lines? Why we need brackets around spaces?